.github/workflows/release.yaml

```
@@ -0,0 +1,33 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install Dependencies
+        run: yarn
```

I see a `package-lock.json` file, I guess it will be better if we use the `npm ci` command instead of `yarn` here, no?

```suggestion
        run: npm ci
```

---

Very much so :)